experimental/dds/tree2/src/test/snapshots/testTrees.ts

```diff
@@ -58,7 +81,8 @@ function generateTreeRecursively(
 			if (height === 1) {
 				const writeCursor = singleTextCursor({
 					type: leafNodeSchema.name,
-					// TODO: these values show up in the snapshot as "[object Object]", which doesn't seem right.
+					// TODO: these values show up in the snapshot as "[object Object]",
```

Comment:
this comment appears to no longer be true, i'd just remove it.

it would be a problem if it were true.